src/wallet/wallet2.cpp

```diff
@@ -8610,8 +8608,31 @@ wallet2::request_stake_unlock_result wallet2::can_request_stake_unlock(const cry
       }
     }
 
-    add_service_node_pubkey_to_tx_extra(result.ptx.tx.extra, sn_key);
-    add_tx_key_image_unlock_to_tx_extra(result.ptx.tx.extra, unlock);
+    std::vector<cryptonote::tx_destination_entry> dsts;
+    cryptonote::tx_destination_entry de = {};
+    de.addr = this->get_address();
+    de.is_subaddress = false;
+    de.amount = 0;
+    dsts.push_back(de);
+
+    std::optional<uint8_t> hf_version = get_hard_fork_version();
+    std::vector<uint8_t> extra;
+    uint32_t priority = this->get_default_priority();
```

Comment:
Priority here should always be 1 (unimportant) because this TX isn't one that should be blinked (in particular, it does not take effect until processed in a block, so the blink priority makes little sense).